TrackMyStuffApp/appsettings.json

```diff
@@ -9,7 +9,15 @@
         "Args": {
           "path": "Logs/log-.txt",
           "rollingInterval": "Day",
-          "buffered":  true
+          "buffered": true
+        }
+      },
+      {
+        "Name": "Graylog",
+        "Args": {
+          "hostnameOrAddress": "localhost",
+          "port": "12201",
+          "transportType": "Udp"
```

Comment:
To send log to Graylog, we will need to:

- Install [Docker desktop](https://www.docker.com/products/docker-desktop).
- Run these commands in the command line to install GrayLog locally.:

```sh
docker run --name mongo -d mongo:4.2


docker run --name elasticsearch \
    -e "http.host=0.0.0.0" \
    -e "discovery.type=single-node" \
    -e "ES_JAVA_OPTS=-Xms512m -Xmx512m" \
    -d docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.0

docker run --name graylog --link mongo --link elasticsearch \
    -p 9000:9000 -p 12201:12201/udp -p 1514:1514 \
    -e GRAYLOG_HTTP_EXTERNAL_URI="http://127.0.0.1:9000/" \
    -d graylog/graylog:4.0

```

**Note**: we set the `/udp` in  `-p 12201:12201/udp` because that's how appsettings.json is setup.


- Once install, please access Graylog via this link: [http://127.0.0.1:9000/system/inputs](http://127.0.0.1:9000/system/inputs).
- We need to create a new **input** to listen and fetch log from the TrackMyStuff application.
- When create a new **input**, please use `GELF UDP` and with `12201` as the port number.